MyFirstSlideshow/app/src/main/res/layout/activity_main.xml

```diff
@@ -4,11 +4,87 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     tools:context="com.charles.myfirstslideshow.MainActivity">
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical">
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Hello World!"
-        />
+        <ImageView
+            android:layout_width="match_parent"
+            android:layout_height="400px"
```

Comment:
activity_main : line 14 - height also should be defined in the dimens.xml

Reply:
activity_main : line 14 - if you are making width match_parent there are so many chances that image can be stretch/scale. so have to handle it
and height should be specify in res/dimens.xml